ui/widgets/resizable.js

```diff
@@ -381,20 +385,25 @@ $.widget( "ui.resizable", $.ui.mouse, {
 		this.offset = this.helper.offset();
 		this.position = { left: curleft, top: curtop };
 
+		var calculatedSize = undefined;
```

Comment:
Please move the declaration to the single var we have above, adding it as a first entry in its own line, without the `undefined` initializer - that's our code style:
```js
		var calculatedSize,
			scroll = ( a && a === "left" ) ? "scrollLeft" : "scrollTop",
			has = false;
```